lib/kast.js

```diff
@@ -282,6 +283,8 @@ const format = (o, isRaw = false, mixFix = false, colorize = false) => {
       "intList" : "",
       ".List{\"intList\"}": "",
       ".WordStack_EVM-DATA": "",
+      "#WordPackUInt112UInt112UInt32(_,_,_)_RULES": "#WordPackUInt112UInt112UInt32",
+      "_[_:=_]_EVM-DATA": "EVM-DATA",
```

Comment:
I'm not sure this is a good idea anymore. `EVM-DATA` is the module name, it make no sense to remove the rest and leave this in.

Maybe this should be prettified as `[ := ]`?